staging/src/k8s.io/apiextensions-apiserver/test/integration/conversion_test.go

```diff
@@ -131,7 +133,8 @@ func TestWebhookConverter(t *testing.T) {
 
 	for _, test := range tests {
 		t.Run(test.group, func(t *testing.T) {
-			tearDown, webhookClientConfig, webhookWaitReady, err := convert.StartConversionWebhookServerWithWaitReady(test.handler)
+			upCh, handler := closeOnCall(test.handler)
+			tearDown, webhookClientConfig, err := convert.StartConversionWebhookServer(handler)
```

Comment:
@jpbetz I got rid of StartConversionWebhookServerWithWaitReady. Composing this here is simpler and more go-like, using composition.